src/containers/guest-home-page/how-it-works/HowItWorks.js

```diff
@@ -0,0 +1,61 @@
+import { Box } from '@mui/system'
+import { Typography } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import Stack from '@mui/material/Stack'
+import Switch from '@mui/material/Switch'
+import { useState } from 'react'
+import ShareYourExperience from '~/containers/share-your-experience/ShareYourExperience'
+import LearnFromExperts from '~/containers/learn-from-experts/LearnFromExperts'
+
+const style = {
+  block: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    pb: '80px'
+  },
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column'
+  }
+}
+
+const HowItWorks = () => {
+  const { t } = useTranslation()
+
+  const [isStudent, setIsStudent] = useState(false)
+
+  const handleChange = () => {
+    setIsStudent(!isStudent)
+  }
+
+  const cardMap = isStudent ? <ShareYourExperience /> : <LearnFromExperts /> 
+
+  return (
+    <Box sx={ style.block }>

```

Comment:
add id to this section like in WhatCanYouDo  and WhoWeAre

Reply:
added